redhat-developer/developers.redhat.com
# [RHDX-250] Adobe DDO Taxonomy

### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-250

### Related Docs

https://mojo.redhat.com/docs/DOC-1093367

### Verification Process

* We can verify that we are passing taxonomy terms (by label(s)) properly to Adobe via the digitalData JS object
*  We can verify that these taxonomy terms are being captured by Adobe and can be used to filter data by our Data Analysts

For each of the following scenarios/pages, do this:

* Visit the URL
* Open your Chrome console
* Refresh (Shift+Cmd+R)
* Type `digitalData.page.attributes` or just `digitalData` into the console to inspect the digitalData JS object. We are concern with key/values under `digitalData.page.attributes` specifically.

You should see this:

* Each of the `digitalData.page.attributes.taxonomy*` key/values should match the taxonomy terms referenced by the respective taxonomy field of that node
* If a taxonomy field does not reference any term, then an empty array is set

#### OpenJDK (new, assembly-based Product page)

https://developer-preview-3515.ext.us-west.dc.preprod.paas.redhat.com/products/openjdk/overview

<img width="1676" alt="PR3515-ProductsOpenjdkOverview" src="https://user-images.githubusercontent.com/7155034/75469050-27955380-5943-11ea-9139-dbd865bad2b8.png">

#### RHEL (old, paragraphs-based Product page)

https://developer-preview-3515.ext.us-west.dc.preprod.paas.redhat.com/products/rhel/overview

<img width="1674" alt="PR3515-ProductsRhelOverview" src="https://user-images.githubusercontent.com/7155034/75469055-2a904400-5943-11ea-93a1-ae7194a88708.png">

#### Kubernetes Topic Page

https://developer-preview-3515.ext.us-west.dc.preprod.paas.redhat.com/topics/kubernetes/

<img width="1673" alt="PR3515-TopicsKubernetes" src="https://user-images.githubusercontent.com/7155034/75469062-2d8b3480-5943-11ea-85df-f2a3ad983ed4.png">

#### Containers Topic Page

https://developer-preview-3515.ext.us-west.dc.preprod.paas.redhat.com/topics/containers/

<img width="1676" alt="PR3515-TopicsContainers" src="https://user-images.githubusercontent.com/7155034/75469067-30862500-5943-11ea-9c6d-4ba1139c81f1.png">
